include/wx/webview.h

```diff
@@ -151,19 +152,22 @@ class WXDLLIMPEXP_WEBVIEW wxWebView : public wxControl
 
     static void RegisterFactory(const wxString& backend,
                                 wxSharedPtr<wxWebViewFactory> factory);
+    static bool IsBackendAvailable(const wxString& backend);
 
     // General methods
     virtual void EnableContextMenu(bool enable = true)
     {
         m_showMenu = enable;
     }
+    virtual void EnableDevTools(bool WXUNUSED(enable) = true) { }
```

Comment:
Oh, sorry for being annoying, but I forgot to mention that if we rename the accessor, we probably should rename this one to `EnableAccessToDevTools()` too, just for symmetry. It's not very important, of course, but IMHO it's still worth doing. If you'd like, I can do the renaming myself when merging.